src/System.Private.CoreLib/shared/System/IO/MemoryStream.cs

```diff
@@ -547,6 +547,51 @@ public override Task CopyToAsync(Stream destination, int bufferSize, Cancellatio
             }
         }
 
+        public override void CopyTo(Buffers.ReadOnlySpanAction<byte, object?> callback, object? state, int bufferSize)
+        {
+            StreamHelpers.ValidateCopyToArgs(this, callback, bufferSize);
```

Comment:
This can be moved to after the `return base.` call.  No need to duplicate the check.

Reply:
Fixed.